src/appleseed.studio/mainwindow/rendering/lightpathswidget.cpp

```diff
@@ -213,7 +209,7 @@ namespace
         void end_object() override
         {
             glEnd();
-        };
+        }
```

Comment:
Wow, that compiled?!

Reply:
clang emits a warning. All other compilers accept it happily.